ingest/templates/datawave-ingest-config-pvc.yaml

```diff
@@ -0,0 +1,12 @@
+{{ if (.Values.config.sharedConfig).create }}
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: {{ .Values.config.sharedConfig.claim }}
+spec:
```

Comment:
I feel like we should define spec.storageClassName and make it a value so we arent relying on the default, for the case that we end up in a cluster with multiple scn options.